rootfs/templates/confd_settings.py

```diff
@@ -16,6 +16,8 @@
 
 {{ if exists "/deis/controller/registrationMode" }}
 REGISTRATION_MODE = '{{ getv "/deis/controller/registrationMode" }}'
+{{ else }}
+REGISTRATION_MODE = 'enabled'
```

Comment:
I switched `confd` to `env` mode so what you proposed and what's there are pretty much the same. I didn't want to try to replace `confd` yet for the `gunicorn` config